packages/design-system/src/components/popup/utils/getOffset.js

```diff
@@ -152,7 +142,7 @@ export function getOffset({
 
   // Clamp values
   return {
-    x: Math.max(0, Math.min(offsetX(), maxOffsetX)),
+    x: Math.max(0, Math.min(offsetX, maxOffsetX)),
```

Comment:
I decided against grabbing this from the units package since we don't already have that package as a dep for the design system package. Seemed like a lot to pull in for this one simple func. 🤷